lib/data/remote/creators_dto.dart

```diff
@@ -12,20 +12,20 @@ class Creators {
     if (json['items'] != null) {
       items = <Items>[];
       json['items'].forEach((v) {
-        items!.add(new Items.fromJson(v));
+        items!.add(Items.fromJson(v));
```

Comment:
Is this bang operator (!) somehow needed here? Please avoid using non-null assertions, it's always safer to add null check